libnavigation-copilot/src/test/java/com/mapbox/navigation/copilot/MapboxCopilotImplTest.kt

```diff
@@ -2420,6 +2422,110 @@ class MapboxCopilotImplTest {
         }
     }
 
+    @Test
+    fun `two active guidance sessions stop and finish, computation dispatcher doesn't for the first session`() {
```

Comment:
`doesn't` what? run? It seems a typo 😅

Reply:
Thanks. Replaced by `two active guidance sessions are started and stopped, computation dispatcher doesn't work during the first session`.